### PR TITLE
Add OSSE behaviour to COR and Demographic Updates

### PR DESCRIPTION
**Pull request by TreyE**

### CODE REVIEW

File: app/models/enrollment_action/reinstatement_comparison_helper.rb [hunk 1]
```diff
@@ -77,8 +79,9 @@ def ivl_reinstatement_candidates(policy_cv)
       end
     end
 
-    def ivl_reinstatement_candidate?(pol, plan, subscriber_id, subscriber_start)
+    def ivl_reinstatement_candidate?(pol, plan, subscriber_id, subscriber_start, is_osse)
```
Comment (nks2109) [inline app/models/enrollment_action/reinstatement_comparison_helper.rb:82]: This method has been used in the above method `ivl_reinstatement_candidates(policy_cv)` and it is not defined the is_osse parameter

File: app/models/enrollment_action/reinstatement_comparison_helper.rb [hunk 2]
```diff
@@ -48,13 +48,15 @@ def shop_reinstatement_candidates(policy_cv)
...
       plan = extract_plan(policy_cv)
+      is_osse = is_osse?(policy_cv)
       subscriber_person.policies.select do |pol|
-        shop_reinstatement_candidate?(pol, plan, employer, subscriber_id, subscriber_start)
+        shop_reinstatement_candidate?(pol, plan, employer, subscriber_id, subscriber_start, is_osse)
       end
     end
 
-    def shop_reinstatement_candidate?(pol, plan, employer, subscriber_id, subscriber_start)
+    def shop_reinstatement_candidate?(pol, plan, employer, subscriber_id, subscriber_start, is_osse)
```
Comment (nks2109) [inline app/models/enrollment_action/reinstatement_comparison_helper.rb:57]: This method has been used in the above method ivl_reinstatement_candidates(policy_cv) and it is not defined `is_osse` parameter

**Review by nks2109 — CHANGES_REQUESTED** [review]

File: app/models/enrollment_action/reinstatement_comparison_helper.rb [hunk 3]
```diff
@@ -48,13 +48,15 @@ def shop_reinstatement_candidates(policy_cv)
       plan_year = find_employer_plan_year(policy_cv)
```
Comment (nks2109) [inline app/models/enrollment_action/reinstatement_comparison_helper.rb:48]: even this plan_year can be removed

Comment (TreyE) [reply app/models/enrollment_action/reinstatement_comparison_helper.rb:48]: I honestly don't know why this was ever even in the code, it just would have always returned nil.

**Review by nks2109 — APPROVED** [review]

**Pull request merged**